builtin/logical/aws/path_config_root_test.go

```diff
@@ -31,6 +31,8 @@ func TestBackend_PathConfigRoot(t *testing.T) {
 		"iam_endpoint":            "https://iam.amazonaws.com",
 		"sts_endpoint":            "https://sts.us-west-2.amazonaws.com",
 		"sts_region":              "",
+		"sts_fallback_endpoints":  []string{},
+		"sts_fallback_regions":    []string{},
```

Comment:
Are there more test cases we can add for this?

Reply:
added a couple tests cases - normal input, comma separated string, and mismatched entry error check.